cvat-ui/src/components/quality-control/task-quality/allocation-table.tsx

```diff
@@ -32,6 +33,55 @@ interface RowData {
     active: boolean;
 }
 
+interface TableRowData extends RowData {
+    key: Key;
+}
+
+export function getAllocationTableContents(gtJobMeta: FramesMetaData, gtJob: Job): TableRowData[] {
```

Comment:
Let's at least add comment that this function is going to be used of application code by plugins.